.github/workflows/frontend-check.yaml

```diff
@@ -9,13 +9,15 @@ jobs:
   install-podman:
     runs-on: ubuntu-latest
     steps:
-      - name: install podman
-        run: apt-get -y install podman
+      # - name: install podman
+      #   run: apt-get -y install podman
       - name: Checkout
         uses: actions/checkout@v4
       - name: run pod composed e2e gui tests
-        run: ./pod-e2e.sh
-        
+        run: |
+          alias podman=docker
+          ./pod-e2e.sh
```

Comment:
Why are you running `docker` commands? That shouldn't be necessary ... I was able to use `podman` commands directly, and I'd rather not involve `docker` if we don't have to...

Reply:
I assumed `podman` was not available in the ci job environment. I can change it.